src/keybindings.h

```diff
@@ -274,6 +274,7 @@ enum GeanyKeyBindingID
 	GEANY_KEYS_FORMAT_SENDTOCMD8,				/**< Keybinding. */
 	GEANY_KEYS_FORMAT_SENDTOCMD9,				/**< Keybinding. */
 	GEANY_KEYS_EDITOR_DELETELINETOBEGINNING,	/**< Keybinding. */
+	GEANY_KEYS_GOTO_ADD_POSITION,				/**< Keybinding. */
```

Comment:
I think it should follow existing style and if desired, to add a separate commit renaming all of the enumerators later.